.github/workflows/deploy-gcp.yml

```diff
@@ -15,15 +15,15 @@ on:
         description: Runtime version
         required: true
         type: string
-        default: python312
```

Comment:
As Cloud Functions doesn't support Python 3.13 runtime yet, we should keep using Python 3.12 for now: https://cloud.google.com/functions/docs/runtime-support#python

Reply:
Thanks for the update!